src/scripting/lua_map.cc

```diff
@@ -1933,6 +1938,31 @@ int LuaTribeDescription::get_resource_indicators(lua_State* L) {
 	return 1;
 }
 
+/* RST
+   .. attribute:: collectors_points_table
+
+      (RO) the ``collectors_points_table`` as defined in the tribe's `units.lua`.
+      See `data/tribes/initializations/atlanteans/units.lua` for more information
+      on the table structure.
+*/
+int LuaTribeDescription::get_collectors_points_table(lua_State* L) {
```

Comment:
Where is this function used? The collectors table is defined in lua, so a get function shouldn't be needed?

Reply:
Lua unit definitions are not persistent. The table defining the tribe is parsed once during tribe loading and then discarded, it's just a local variable passed to the `descriptions:new_tribe` function. All such information needs to be stored in the C++ `TribeDescr` objects and made available to scripts via the `LuaTribeDescription`.

Reply:
Thanks for the explanation.